src/components/nav.js

```diff
@@ -2,10 +2,10 @@ import { Link } from 'react-router-dom';
 
 const Nav = () => (
   <div className="nav">
-    <span>Bookstore CMS</span>
+    <span className="cms">Bookstore CMS</span>
     <nav>
-      <Link to="/">Book</Link>
-      <Link to="/categories">Categories</Link>
+      <Link className="nav-book" to="/">Book</Link>
+      <Link className="nav-category" to="/categories">Categories</Link>
     </nav>
   </div>
 );
```

Comment:
- [ ] Please stick to the Zaplin design you missed the user icon on the right side if you check the Zaplin design there is one user icon on the right side but in your case, you missed this icon so kindly add this icon as well :slightly_smiling_face: 
- [ ] If you check Zaplin design's navbar you will see there is border on the bottom side but in your case this border is not there so kindly fix this issue as well :+1: 

**Your Design**
![Screenshot from 2023-08-24 00-31-49](https://github.com/As1imwe-Mark/book_store/assets/45226848/fa16fe26-e4f4-4247-8fde-e8154eb5f084)

**Zaplin Design**
![Screenshot from 2023-08-24 00-30-18](https://github.com/As1imwe-Mark/book_store/assets/45226848/0c72faa4-ab73-461c-a7e7-ab5bb2c7fbcd)